broker-daemon/bin/sparkswapd.js

```diff
@@ -5,81 +5,80 @@
+  // Broker configuration options
+  .option('--broker.data-dir [data-dir]', 'Location to store SparkSwap data', validations.isFormattedPath, config.broker.dataDir)
+  .option('--broker.interchain-router-address [interchain-router-address]', 'Add a host/port to listen for interchain router RPC connections', validations.isHost, config.broker.interchainRouterAddress)
+  .option('--broker.id-pub-key [id-pub-key]', 'Location of the public key for the broker\'s identity', validations.isFormattedPath, config.broker.idPubKey)
+  .option('--broker.id-priv-key [id-priv-key]', 'Location of private key for the broker\'s identity', validations.isFormattedPath, config.broker.idPrivKey)
+  // TODO: Make this a list instead of string
+  .option('--broker.markets [markets]', 'Comma-separated market names to track on startup', validations.areValidMarketNames, config.broker.markets)
+  .option('--broker.disable-auth [disable-auth]', 'Disable SSL for the broker (DEV ONLY)', program.BOOL, config.broker.disableAuth)
+  // Relayer configuration options
+  .option('--relayer.rpc-host [rpc-host]', 'The host address for the SparkSwap Relayer', validations.isHost, config.relayer.rpcHost)
+  .option('--relayer.cert-path [cert-path]', 'Location of the root certificate for the SparkSwap Relayer (only used in development)', validations.isFormattedPath, config.relayer.certPath)
+  // Broker RPC configuration options
+  .option('--rpc.rpc-address [rpc-rpc-address]', 'Add a host/port to listen for daemon RPC connections', validations.isHost, config.rpc.rpcAddress)
+  .option('--rpc.rpc-user [rpc-user]', 'Broker rpc user name', program.String, config.rpc.rpcUser)
+  .option('--rpc.rpc-pass [rpc-pass]', 'Broker rpc password', program.String, config.rpc.rpcPass)
+  .option('--rpc.rpc-pub-key [path]', 'Location of the public key for the broker\'s rpc', validations.isFormattedPath, config.rpc.rpcPubKey)
+  .option('--rpc.rpc-priv-key [path]', 'Location of private key for the broker\'s rpc', validations.isFormattedPath, config.rpc.rpcPrivKey)
 
+// For each currency, we will add expected options for an engine
 for (let currency of currencies) {
   let lowerSymbol = currency.symbol.toLowerCase()
-  let ENGINE_TYPE = process.env[`${currency.symbol}_ENGINE_TYPE`]
-  let LND_TLS_CERT = process.env[`${currency.symbol}_LND_TLS_CERT`]
-  let LND_MACAROON = process.env[`${currency.symbol}_LND_MACAROON`]
-  let LND_RPC_HOST = process.env[`${currency.symbol}_LND_RPC_HOST`]
+
   program
-    .option(`--${lowerSymbol}-engine-type <type>`, `The type of underlying Payment Channel Network node for ${currency.name}`, [ 'LND' ], ENGINE_TYPE)
+    .option(`--${lowerSymbol}.engine-type [engine-type]`, `The type of underlying Payment Channel Network node for ${currency.name}`, isSupportedEngineType, currency.engineType)
+    .option(`--${lowerSymbol}.rpc-host [rpc-host]`, `Location of the LND RPC server to use for ${currency.name}.`, validations.isHost, currency.rpcHost)
```

Comment:
don't you want to group this under an lnd-specific command, by naming it something like `--${lowerSymbol}.lnd-rpc-host` ?

Reply:
@treygriffith I removed the lnd prefix because rpc-host should be pretty generic for us. However I should prefix the macaroon tag w/ `lnd`

Reply:
I might change the description for the generic ones to be more generic then